src/types/summary_helper.rs

```diff
@@ -1236,6 +1252,10 @@ fn convert_account_transaction(
                         delegator_id,
                         account: _,
                     } => Ok(super::DelegationEvent::DelegationRemoved { delegator_id }),
+                    Event::BakerRemoved {
```

Comment:
Would it be possible to rename `baker` to `validator` and `baker_id` to `validtor_id` since the events are publicly facing? I see it would need the update of the whole code base and might not be the right place to do it now.

Reply:
That could be worth doing, but it's out of scope for this, and I'd rather maintain consistency with the naming within the SDK than start sneaking in `validator`s. If we do such a renaming across the API, it will be a major breaking change, so we should consider if it's worthwhile to do it.